### DISCUSSION TITLE
When I delete the only plot, then generate new one - will it be different?

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#1701] [Q&A]
My question is about scenario, where I generate plots incrementally on one machine and move them to the other, with deletion in order to make room for new ones. So, when I start again generating on the same machine with the same keys: will it generate the same plots?

**Comment by hoffmang9** [comment]
Your question is a Frequently Asked Question answered right here on the GitHub wiki - https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#can-i-make-plots-on-one-machine-and-move-it-to-another-machine

**Reply by ardabro** [reply]
But this information doesn't clarify the question of re-generating plots using the same keys after deletion of old plots. And this is essential. If it starts generating exactly the same plots it is useless and one should rather generate new keys for each iteration.

**Comment by dorkmo** [comment]
every plot is unique from the begining
